src/sql/workbench/contrib/notebook/browser/models/modelInterfaces.ts

```diff
@@ -413,6 +417,85 @@ export interface INotebookModel {
 
 	standardKernels: IStandardKernelWithProvider[];
 
+	getFindCount(): number;
+
+	getFindIndex(): number;
+
+	findNext(): Thenable<NotebookRange>;
```

Comment:
Is there a reason you need to use Thenable here? Just using Promise directly is preferred